include/click/packet.hh

```diff
@@ -837,6 +842,10 @@ class Packet { public:
 #if !CLICK_PACKET_USE_DPDK
 	char timestamp[sizeof(Timestamp)];
 #endif
+#if INLINED_ALLANNO
```

Comment:
Why do we have to change the name?

Reply:
Compile error! :-p It seems we cannot have an inline function and a variable with the same name.